src/Command/CommandBase.php

```diff
@@ -1089,7 +1089,7 @@ public static function getAliasCache() {
    */
   protected function doGetApplicationFromAlias($application_alias) {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
-    $acquia_cloud_client->addQuery('filter', 'hosting=@*' . $application_alias);
+    $acquia_cloud_client->addQuery('filter', 'hosting=prod:' . $application_alias);
```

Comment:
Sometimes this wildcard causes multiple applications to be returned. E.g., when an alias is `blue` and there are apps `prod:greenblue, prod:veryblue, prod:blue`, then `prod:greenblue` may be returned when `blue` is requested.

Reply:
Trouble is this may break ACSF sites. Not sure those actually start with `prod:`

Reply:
Yeah, possible prefix values are devcloud, gardens, prod, network, enterprise-g1.